include/drivers/led.h

```diff
@@ -22,6 +22,23 @@
 #include <zephyr/types.h>
 #include <device.h>
 
+/**
+ * @brief LED information structure
+ *
+ * This structure gathers useful information about LED controller.
+ *
+ * @param label LED label.
+ * @param num_colors Number of colors per LED.
+ * @param index Index of the LED on the controller.
+ * @param color_mapping Mapping of the LED colors.
+ */
+struct led_info {
```

Comment:
API meeting 11th August 2020:

Since this information is taken directly from Devicetree in this particular driver implementation, we were wondering what the purpose of this API is. We guessed that this might be useful in order to write an application that is supposed to work on multiple boards and that deals with LED controllers, replicating perhaps a sequence of colors without having to refer to the Devicetree source (which may be different for each board/controller). Is that the intent? If so, it would perhaps also be useful to demonstrate this in the sample you added.